src/openrct2/actions/RideCreateAction.cpp

```diff
@@ -148,6 +148,10 @@ GameActions::Result RideCreateAction::Execute() const
         station.Exit.SetNull();
         station.TrainAtStation = RideStation::NO_TRAIN;
         station.QueueTime = 0;
+        station.SegmentLength = 0;
+        station.QueueLength = 0;
+        station.Length = 0;
+        station.Height = 0;
```

Comment:
What about
https://github.com/OpenRCT2/OpenRCT2/blob/9d77219b5d9cd3b5e2854e097f3e37ca782f3723/src/openrct2/ride/Ride.h#L55
https://github.com/OpenRCT2/OpenRCT2/blob/9d77219b5d9cd3b5e2854e097f3e37ca782f3723/src/openrct2/ride/Ride.h#L60
https://github.com/OpenRCT2/OpenRCT2/blob/9d77219b5d9cd3b5e2854e097f3e37ca782f3723/src/openrct2/ride/Ride.h#L63
?

Reply:
They are not relevant for this bug and are actually reset later.